java/client/src/main/java/glide/api/commands/ServerManagementClusterCommands.java

```diff
@@ -315,4 +315,169 @@ public interface ServerManagementClusterCommands {
      * }</pre>
      */
     CompletableFuture<ClusterValue<Long>> lastsave(Route route);
+
+    /**
+     * Displays a piece of generative computer art and the Redis version.<br>
+     * The command will be routed to a random node.
+     *
+     * @see <a href="https://redis.io/commands/lolwut/">redis.io</a> for details.
+     * @return A piece of generative computer art along with the current Redis version.
+     * @example
+     *     <pre>{@code
+     * String data = client.lolwut().get();
+     * System.out.println(data);
+     * assert data.contains("Redis ver. 7.2.3");
+     * }</pre>
+     */
+    CompletableFuture<String> lolwut();
+
+    /**
+     * Displays a piece of generative computer art and the Redis version.<br>
+     * The command will be routed to a random node.
+     *
+     * @see <a href="https://redis.io/commands/lolwut/">redis.io</a> for details.
+     * @param parameters Additional set of arguments in order to change the output.
+     * @return A piece of generative computer art along with the current Redis version.
+     * @example
+     *     <pre>{@code
+     * String data = client.lolwut(new int[] { 40, 20 }).get();
```

Comment:
lolwut without version just ignores all rest parameters, they could be even strings

Reply:
I think that is true when running against Redis 7 because they don't generate any art for that version. If I run lolwut against Redis 6 with no version but these two parameters specified I can see it changes the output.